src/lib/drivers/adin2111/src/adi_hal.c

```diff
@@ -1,113 +1,21 @@
-/*
- *---------------------------------------------------------------------------
- *
- * Copyright (c) 2020, 2021 Analog Devices, Inc. All Rights Reserved.
- * This software is proprietary to Analog Devices, Inc.
- * and its licensors.By using this software you agree to the terms of the
- * associated Analog Devices Software License Agreement.
- *
- *---------------------------------------------------------------------------
- */
-
-/** @addtogroup hal HAL API
- *  @{
- */
-
-#include <string.h>
 #include "adi_hal.h"
 #include "adi_bsp.h"
+#include "bsp.h"
+#include <string.h>
 
-/* Lookup table of CRC32 constants */
-static const uint32_t crc32_tab[] = {
-	0x00000000, 0x77073096, 0xEE0E612C, 0x990951BA, 0x076DC419, 0x706AF48F,

```

Comment:
This is in a separate adi_fcs.c in the bmcore adin driver. Other parts of this file are the locally required abstraction implementations though.